tasks/watch.cr

```diff
@@ -166,15 +166,16 @@ module LuckySentry
     end
 
     private def print_error_message
-      puts "There was a problem compiling. Watching for fixes...".colorize(:red)
       if successful_compilations.zero?
         puts <<-ERROR
 
-        Try this...
+        #{"---".colorize.dim}
```

Comment:
Divider so it is separate from the Crystal compile error

Reply:
ooo nice